src/views/PeopleSearch/index.js

```diff
@@ -386,8 +386,8 @@ class PeopleSearch extends Component {
 
   async updateURL() {
     const searchParameters = Object.entries(this.state.searchValues)
-      .filter((n) => n) // removes empty strings
       .map(([key, value]) => (value ? `${key}=${value}` : '')) // [ 'firstName=value', 'state=texas']
+      .filter((n) => n) // removes empty strings
```

Comment:
We can do the filtering first, you just have to access  the value part of the [key, value] tuple:
```suggestion
      .filter(([key, value]) => value)
      .map(([key, value]) => (value ? `${key}=${value}` : ''))
```

That would also allow you to simplify the `map` call, since all the empty searchValues have already been removed:
```suggestion
      .filter(([key, value]) => value)
      .map(([key, value]) => (`${key}=${value}`))
```